docs/src/basics.md

```diff
@@ -35,6 +35,12 @@ fuelup self update
 ```
 <!-- update_fuelup:example:end -->
 
+## Using Http Proxy
+
+To configure `fuelup` to use your proxy setting you can change `http_proxy` environment value. The value format is in [libcurl format](https://everything.curl.dev/libcurl/proxies#proxy-types) as in `[protocol://]host[:port]`.
+
+***Warning: don't leave `http_proxy` environment with empty string or other invalid format***
```

Comment:
We can add other options here as well since they are added with #592

Reply:
@kayagokalp Suggestion changes addressed~ Thanks for your review